### PR TITLE
JO settings icon: UI

### PR DESCRIPTION
**Pull request by GabisPapin**
-created popup with settings button and logout button in navbar menu;
-add styles in the popup;
-the settings button redirect to settings page;
-the logout clean persist infos and store, and redirect to the sign in page.

### CODE REVIEW

**Comment by GabisPapin** [comment]
file:///home/gabrielapapin/Imagens/JO-popup-settings.png

**Comment by GabisPapin** [comment]
JO-popup-settings

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_GOOGLE_REDIRECT=http://localhost:3000/auth/redirect
```
Comment (lyudad) [inline .env:1]: env file should be in gitignore!!!!

File: src/components/Layout/Layout.tsx [hunk 2]
```diff
@@ -42,7 +52,7 @@ const Layout: FC = () => {
 		};
 	}, []);
 
-	const handleClick = () => {
+	const handleClick = async () => {
 		dispatch(saveEmail(''));
 		dispatch(saveToken(''));
 		dispatch(saveUserId(0));
```
Comment (lyudad) [inline src/components/Layout/Layout.tsx]: I dont like 0 in saveUserId. can we set null?

Comment (GabisPapin) [reply src/components/Layout/Layout.tsx]: ok, I just testing and forget to return the correct value. I'll fix it.

**Pull request merged**